api-extractor/src/definitions/ApiField.ts

```diff
@@ -0,0 +1,25 @@
+import * as ts from 'typescript';
+import { ApiItemKind, IApiItemOptions } from './ApiItem';
+import ApiMember from './ApiMember';
+
+/**
+ * This class is part of the ApiItem abstract syntax tree. It represents fields of
+ * an ApiNamespace.
+ */
+class ApiField extends ApiMember {
+  public type: string;
+  public name: string;
+  public value: string;
+
+  constructor(options: IApiItemOptions) {
+    super(options);
+    this.kind = ApiItemKind.Field;
+
+    const propertySignature: ts.PropertySignature = options.declaration as ts.PropertySignature;
+    this.type = propertySignature.type.getText();
+    this.name = propertySignature.name.getText();
+    this.value = propertySignature.initializer.getText(); // value of the export
```

Comment:
This isn't allowed in typescript, "[ts] 'const' declarations must be initialized". If the variable declaration is not 'const', then our system will report a warning, at which time the the compiler will require the const to be initialized with a value.